pokecli.py

```diff
@@ -64,7 +64,7 @@
 try:
     import pkg_resources
     pgoapi_version = pkg_resources.get_distribution("pgoapi").version
-    if pgoapi_version < '1.1.8':
+    if pgoapi_version != '1.1.6':
```

Comment:
The bot was using my api fork at https://github.com/joelgreen/pgoapi which i incremented the version number, however since switching the changes i made have become obsolete so I'm suggesting we switch back to the main branch which is now about 30 commits ahead. However they never incremented the version number.